search-provider/eks-discovery-feed-provider.c

```diff
@@ -711,6 +644,16 @@ handle_artwork_card_descriptions (EksDiscoveryFeedDatabaseContentProvider *skele
       return TRUE;
 
     EkncEngine *engine = eknc_engine_get_default ();
+    g_autoptr(GError) local_error = NULL;
+
+    if (!eks_ensure_app_shards_are_symlinked_to_home_directory (engine,
```

Comment:
In fact content from an SDK1 app also shows up in the discovery feed before the app has been run, without this.